apps/finance/app/.babelrc

```diff
@@ -5,13 +5,24 @@
       {
         "modules": false,
         "useBuiltIns": "entry",
-        "core-js": 3,
-        "shippedProposals": true,
+        "corejs": 3,
```

Comment:
🤦‍♂️ all of our configs with corejs are wrong haha.

Do you mind also fixing this in `aragon/aragon`, `aragon-react-boilerplate`, and `your-first-aragon-app` (I think that's all?)?

Reply:
Done for [`aragon/aragon`](https://github.com/aragon/aragon/pull/1453), [`aragon-react-boilerplate`](https://github.com/aragon/aragon-react-boilerplate/pull/112) and [`your-first-aragon-app`](https://github.com/aragon/your-first-aragon-app/pull/19).